src/virtual_maize_field/world_generator/field.world.template

```diff
@@ -52,16 +52,6 @@
       <real_time>0</real_time>
       <wall_time>0</wall_time>
       <iterations>0</iterations>
-      <model name='ground_plane'>
```

Comment:
By removing the ground plane, you will trigger an error in Gazebo, that will print, that there is no ground plane. Otherwise it has no negative effects, I think.

Reply:
On my pc, I don't see an error when removing this part.